### PR TITLE
Update theme to reflect the new logo and brand color changes

### PR DESCRIPTION
**Pull request by tsteur**
We're going to update our logo, see https://matomo.org/blog/2018/10/matomo-logo-reveal/ and https://matomo.org/blog/2018/09/choose-your-favourite-logo/ . Thanks for voting and choosing our new logo!

As the logo contains some new brand colors we will reflect these changes in the Matomo app. Themes should be unaffected by this.

Matomo Mobile app is already updated in https://github.com/matomo-org/matomo-mobile-2/commit/82dca1d1d83a5e4e416ce699cd23a21cec3f424a but not yet released.

### CODE REVIEW

**Comment by Findus23** [comment]
I really like the new theme.

Really quick feedback:
- The color contrast in notifications is really bad:
https://contrast-ratio.com/#%23fff4e3-on-%23f57c00

- `<meta name="theme-color" content="#37474f">` still has the old brand color.

And I think there are a few people who dislike bright colors, so maybe we should publish a really basic theme that sets theme-color-brand back to `#37474f`.

**Comment by tsteur** [comment]
Cheers, fixing the theme color now.

Making the color contrast for warnings a wee bit better but even when choosing a #fff it won't be much better. Would need to change the orange.

Releasing a basic theme should be easily doable by just changing one or two color codes 👍  Could be like "Matomo 2018" or "Matomo colorless" :) For improved contrasts there could be maybe an extra theme as well if there is none already?

File: plugins/Morpheus/templates/layout.twig [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
             {% include "@CoreHome/_favicon.twig" %}
             {% include "@CoreHome/_applePinnedTabIcon.twig" %}
-            <meta name="theme-color" content="#37474f">
+            <meta name="theme-color" content="#43a047">
```
[the remark this answers is not in the document]
Comment (tsteur) [reply plugins/Morpheus/templates/layout.twig]: definitely should #3450A3 👍

[the remark this answers is not in the document]
Comment (tsteur) [reply plugins/Morpheus/templates/layout.twig]: updated

File: plugins/Morpheus/stylesheets/uibase/_headerMessage.less [hunk 2]
```diff
@@ -9,7 +9,7 @@
   border-radius: 4px;
 
   &.update_available:not(.isPiwikDemo) .title {
-    color: #CA8100;
+    color: #fbf7f1;
```
Comment (sgiehl) [inline plugins/Morpheus/stylesheets/uibase/_headerMessage.less]: That seems wrong. 
image

Comment (tsteur) [reply plugins/Morpheus/stylesheets/uibase/_headerMessage.less]: good point, should be `#f57c00`. I'll fix it

**Pull request merged**